<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Смарт</string>
  </property>
  <property name="windowIcon">
   <iconset resource="file.qrc">
    <normaloff>:/newPrefix/ico/bowl.ico</normaloff>:/newPrefix/ico/bowl.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 207);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>571</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>     Добро пожаловать в приложение
             Smart Food Calendar!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>571</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Здесь вы можете следить за балансом нутриентов в своём рационе питания.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>           Давайте знакомиться, я Смарт, мне необходимо кое-что о вас узнать, а именно:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/smart.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(188, 255, 146);
    border-radius: 20px;
  
}
QPushButton:hover {
    background: green; 
}
QPushButton:pressed {
    background: dark;
}</string>
   </property>
   <property name="text">
    <string>Рассказать Смарту о себе</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
